frontend/src/constants/featureFlags.ts

```diff
@@ -2,7 +2,10 @@ import { FeatureFlags } from "src/services/FeatureFlagManager";
 
 // Feature flags should default to false
 export const featureFlags: FeatureFlags = {
-  // This is for showing the search page as it is being developed and user tested
+  // This is for hiding the search page as it is being developed and user tested
   // This should be removed when the search page goes live, before May 2024
-  showSearchV0: true,
+  hideSearchV0: false,
```

Comment:
do we still need this? I changed it to a `hide` vs `show` variable to match the pattern here and allow us to more easily reason about feature flag effects